### DISCUSSION TITLE
Amazing

### DISCUSSION THREAD

**Discussion by FeridQarayev** [FeridQarayev/beautiful_places_app#15] [Q&A] [answered]
What happens ?

**Comment by Ibrahimova11** [comment, marked as answer]
I am writing an assignment, but it is not received.

**Reply by Ibrahimova11** [reply]
And I finally found it

**Reply by Ibrahimova11** [reply]
How are your lessons going?

**Reply by FeridQarayev** [reply]
Good. It's good to talk to Github.

**Reply by Ibrahimova11** [reply]
Exactly. Very interesting

**Reply by FeridQarayev** [reply]
Sometimes it's not clear where the code will end ;)

**Reply by Ibrahimova11** [reply]
Very true
